scripts/build.sh

```diff
@@ -0,0 +1,47 @@
+# Create a browser extension release package
```

Comment:
You'll want to add a shebang here, and adding `set -e` makes it crash completely if there's an error on any line (instead of idiotically continuing like a chicken with its head cut off):

```
#!/bin/bash
set -e
```

Reply:
You're right!